lib/functions/subshell/build_rss.sh

```diff
@@ -31,34 +31,35 @@ cat > $rss <<EOF
     <generator>Arise</generator>
 EOF
 
-[[ $favicon != '' ]] && {
-        cat >> $rss <<EOF
+	[[ $favicon != '' ]] && {
+        cat >> "$rss" <<EOF
     <image>
       <url>$base_url$favicon</url>
       <title>$global_name</title>
       <link>$base_url</link>
     </image>
 EOF
-}
+	}
 
-cat >> $rss <<EOF
+	cat >> "$rss" <<EOF
     <atom:link href="$base_url/rss.xml" rel="self" type="application/rss+xml"/>
         <ttl>60</ttl>
 <lastBuildDate>$(date --rfc-822)</lastBuildDate>
 EOF
 
-# List every directory in our sitemap (except config). This makes up our sitemap since Arise is built to use directory roots as page URLs
-find . -type d -not \( -path ./config -prune \) | while read fname; do
-page_index=$(realpath "$fname"'/index.md')
+	# List every directory in our sitemap (except config). This makes up our sitemap since Arise is built to use directory roots as page URLs
+	find . -type d -not \( -path ./config -prune \) -print0 | \
+		while read -r -d $'\0' fname; do
+			page_index=$(realpath "$fname"'/index.md')
 
-        if [ -e $page_index ]; then
-                get_page_metadata $page_index
+			if [ -e "$page_index" ]; then
+				get_page_metadata "$page_index"
 
-                if [[ $rss_hide != "true" ]] && [[ $is_toc != "true" ]]; then
-                        # Convert html's ISO8601 date to RSS's RFC-822. Fuck you RSS.
-                        rss_date=$(date -d "$published_date" --rfc-822)
-                        
-                        cat >> $rss <<EOF
+				if [[ $rss_hide != "true" ]] && [[ $is_toc != "true" ]]; then
+					# Convert html's ISO8601 date to RSS's RFC-822. stupid RSS.
```

Comment:
look sometimes i just want to employ sentence enhancers for a little whimsy in my comments so that the reader can share in my deranged grievances. compromise? ;)
```suggestion
					# Convert html's ISO8601 date to RSS's RFC-822. sometimes you need a stupid hack to fix an even stupider RFC decision 😐
```